rpc/internal/sps.service.go

```diff
@@ -0,0 +1,74 @@
+package internal
+
+import (
+	"time"
+
+	"github.com/evergreen-ci/poplar"
+)
+
+type TaskInformation struct {
+	Project   string `json:"project"`
+	Version   string `json:"version"`
+	Variant   string `json:"variant"`
+	Order     int    `json:"order"`
+	TaskName  string `json:"task_name"`
+	TaskId    string `json:"task_id"`
+	Execution int    `json:"execution"`
+	Mainline  bool   `json:"mainline"`
+}
+
+type TestInformation struct {
+	TestName  string           `json:"test_name"`
+	Arguments map[string]int32 `json:"arguments"`
```

Comment:
Should trial/parent/tags be in here as well? They're in the poplar model

Reply:
As far as I can tell, those aren't actually used by anything, including Cedar. They still exist in the codebase but for paths that don't seem to be in use. Let me know if that's incorrect though.

Those aren't even being defaulted by SPS like the `Version/type` values below are. I can add it there instead if we just care about that data being present in the database but not about the real values.